pephubclient/models.py

```diff
@@ -1,9 +1,36 @@
-from pydantic import BaseModel
+from typing import Optional
 
+from pydantic import BaseModel, Extra, Field, validator
 
-class JWTDataResponse(BaseModel):
-    jwt_token: str
 
+class ProjectDict(BaseModel):
+    """
+    Project dict (raw) model
+    """
 
-class ClientData(BaseModel):
-    client_id: str
+    description: Optional[str] = ""
```

Comment:
In this case you we can delete `Optional[]`. But the difference between "" and None is: 
When we don't have description and it's none, we want ClientData to return empty string instead of None. e.g.
```
class D1(BaseModel):
     ff: Optional[str] = ""
class D2(BaseModel):
     ff: Optional[str] = None

```
```
>> D1()
DD(ff=:"")

>> D2()
DD(ff=:None)
```

Did I answer your question?